java/ql/src/semmle/code/java/frameworks/spring/SpringUtil.qll

```diff
@@ -0,0 +1,155 @@
+/**
+ * Provides models for the `org.springframework.util` package.
+ */
+import java
+
+/**
+ * Provides models for the `org.springframework.util` package.
+ */
+module SpringUtil {
+  private class FlowSummaries extends SummaryModelCsv {
+    override predicate row(string row) {
+      row = [
+        "org.springframework.util;AutoPopulatingList;false;AutoPopulatingList;(java.util.List<E>, org.springframework.util.AutoPopulatingList$ElementFactory<E>);;Element of Argument[0];Element of ReturnValue;value",
+        "org.springframework.util;AutoPopulatingList;false;AutoPopulatingList;(java.util.List<E>, Class<? extends E>);;Element of Argument[0];Element of ReturnValue;value",
```

Comment:
Signatures won't match. Spaces aren't allowed, and you should use the erased types.